### PR TITLE
LikeButton: update CSS to use flex for alignment.

### PR DESCRIPTION
**Pull request by mtias**
It also sets a min-width on the like number to avoid shuffling the width for single digit increments.

image

----

Before:
image

After:
image

Test live: https://calypso.live/?branch=update/like-button-with-flex

### CODE REVIEW

**Comment by folletto** [comment]
👍

**Comment by blowery** [comment]
the star looks a bit high on Search

woodworking_ _reader_ _wordpress_com

**Comment by blowery** [comment]
should be fixed in https://github.com/Automattic/wp-calypso/pull/7079/commits/b492cf042c1b8cd345ffae77ba7bdde560f0510a

Did notice the comment button still has a 1px bump, and it looks wrong without it. Odd.

**Comment by mtias** [comment]
Good catch.

File: client/components/post-card/style.scss [hunk 1]
```diff
@@ -218,10 +218,6 @@
 	padding-left: 20px;
 }
 
-.post-card__search .like-button .gridicon {
-	top: 2px;
-}
-
 .post-card__search .comment-button .gridicon {
```
Comment (mtias) [inline client/components/post-card/style.scss]: @blowery we probably should remove these as well.

Comment (jancavan) [inline client/components/post-card/style.scss]: @mtias just removed it in: https://github.com/Automattic/wp-calypso/pull/7079/commits/9944436268dd0fc2e1d327023c32a973aefbf059, checked and it didn't make any difference.

**Comment by jancavan** [comment]
LGTM 👍

**Comment by sendhil** [comment]
This blew up the desktop app and is causing every click to have a horrible re-paint:
https://cloudup.com/cSs5t4skbcm

**Comment by sendhil** [comment]
Actually - apologies - had a caching issue with my build and after further testing this isn't the cause for the rendering issue above.

**Pull request merged**